tst/testextra/grplatt.tst

```diff
@@ -193,6 +193,13 @@ gap> Length(ConjugacyClassesSubgroups(SymmetricGroup(7):NoPrecomputedData));
 #I  Using (despite option) data library of perfect groups, as the perfect
 #I  subgroups otherwise cannot be obtained!
 96
+gap> g:=SimpleGroup("Co3");;
```

Comment:
Please do not hesitate to add a new file e.g. `testextra/doublecosets.tst` with this test. Otherwise, the name of the test file and the comment above suggest that this is a test for subgroup lattice routines, and the test does not seem to belong here.  If you think that it does, then at lease prepend it with a couple of lines of comments telling what it tests. This will make easier figuring out the purpose of this test in the future.